mobx/lib/src/api/observable_list.dart

```diff
@@ -263,19 +150,6 @@ class ObservableList<T>
     return _list.toSet();
```

Comment:
Are we discarding the `growable` argument ??

Reply:
yeah, will fix it